parser_library/src/context/variables/system_variable.h

```diff
@@ -59,8 +59,16 @@ class system_variable_sysmac final : public system_variable
 
     // SYSMAC special behavior
     const C_t& get_value(const std::vector<size_t>& offset) const override;
-    const C_t& get_value(size_t idx) const override;
-    const C_t& get_value() const override;
+    const macro_param_data_component* get_data(const std::vector<size_t>& offset) const override;
```

Comment:
https://sonarcloud.io/code?id=eclipse_che-che4z-lsp-for-hlasm&pullRequest=233&selected=eclipse_che-che4z-lsp-for-hlasm%3Aparser_library%2Fsrc%2Fcontext%2Fvariables%2Fsystem_variable.h&line=61

This looks like a valid issue.

Reply:
👍